instances/instance_stop_by_name.go

```diff
@@ -34,14 +30,10 @@ func (c *instancesClient) StopByName(ctx context.Context, name string, drainTime
 		return nil, fmt.Errorf("marshalling request body: %w", err)
 	}
 
-	var response client.ServiceResponse[Instance]
+	var response kcclient.ServiceResponse[StopResponseItem]
 	if err := c.request.DoRequest(ctx, http.MethodPut, Endpoint+"/stop", bytes.NewBuffer(body), &response); err != nil {
 		return nil, fmt.Errorf("performing the request: %w", err)
 	}
 
-	instance, err := response.FirstOrErr()
-	if instance != nil && instance.Message != "" {
-		err = fmt.Errorf("%w: %s", err, instance.Message)
-	}
-	return instance, err
```

Comment:
This comment is for everywhere: I assume that the error message is parsed and passed outside correctly.

Reply:
That's a valid point. Since success and errors responses do not use the same model, we need to think about a way to collect (and return) errors properly with those changes.
Let me think about it and I'll try to propose something to address this problem.

Reply:
@craciunoiuc I restored the error handling in [963d2885..709f67e7](https://github.com/kraftcloud/go-sdk/compare/963d28859a142975843c1dd50db55150cdbe4296..709f67e7346e8145eb4a8838b315b3905f14481c) by embedding a common `client.APIResponseCommon` struct in all response models.